zenodo/modules/deposit/templates/zenodo_deposit/edit.html

```diff
@@ -140,6 +140,34 @@ <h1>Edit upload</h1>
       <h1>New version</h1>
       {%- else %}
       <h1>New upload</h1>
+      {# Note: we need an already initialized deposit #}
+      {% if record.files and current_user_is_admin() %}
+      <div class="btn-group btn-block pull-right">
+        <button type="button" class="btn btn-m btn-default dropdown-toggle pull-right" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Admin <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu">
+          <li><a href="{{ url_for('persistentidentifier.index_view', flt1_21=record.id) }}">
+            Persistent identifiers
+          </a></li>
+          <li><a href="{{ url_for('recordmetadata.details_view', id=record.id) }}">
+            Record
+          </a></li>
+          <li><a href="{{ url_for('bucket.edit_view', id=record.files.bucket.id) }}">
+            Bucket
+          </a></li>
+          <li role="separator" class="divider"></li>
+          <li><a href="{{ url_for('zenodo_deposit.delete', pid_value=pid.pid_value) }}" type="submit">
+            <span class="text-danger"><i class="fa fa-trash"></i> Delete record ...</span>
+          </a></li>
+          {% if record._deposit.created_by %}
+          <li><a href="{{ url_for('zenodo_spam.delete', user_id=record._deposit.created_by) }}" type="submit">
+            <span class="text-danger"><i class="fa fa-trash"></i> Delete user ...</span>
+          </a></li>
+        </ul>
+      </div>
+      {%- endif %}
```

Comment:
```suggestion
          {% if record._deposit.created_by %}
          <li>
            <a href="{{ url_for('zenodo_spam.delete', user_id=record._deposit.created_by) }}" type="submit">
              <span class="text-danger"><i class="fa fa-trash"></i> Delete user ...</span>
            </a>
          </li>
          {%- endif %}
        </ul>
      </div>
```

Otherwise the `<ul>` wouldn't be closed and might mess up the whole page's HTML